tarkov/inventory/inventory.py

```diff
@@ -214,21 +249,30 @@ def transfer(item: Item, to: Item, count: int) -> None:
 
     def fold(self, item: Item, folded: bool) -> None:
         """
-        Folds item
+        Fold or unfold an item.
+
+        :param item: The item to fold/unfold.
+        :param folded: If the item is already folded or not.
```

Comment:
Folded argument determines if item would be folded or not.

Reply:
I'll fix that rn